nominal/core.py

```diff
@@ -709,6 +804,35 @@ def create_video_from_io(
         response = self._ingest_client.ingest_video(self._auth_header, request)
         return self.get_video(response.video_rid)
 
+    def create_log_set(
+        self,
+        name: str,
+        logs: Iterable[Log] | Iterable[tuple[datetime | IntegralNanosecondsUTC, str]],
+        timestamp_type: LogTimestampType = "absolute",
+        description: str | None = None,
+    ) -> LogSet:
+        """
+        Creates a log set, to which logs can be attached using `attach-and-finalize`. The logs within a logset are
+        not searchable until the logset is finalized.
+        Timestamp type must be a string equal to either 'absolute' or 'relative'.
+        """
+        request = datasource_logset_api.CreateLogSetRequest(
+            name=name,
+            description=description,
+            origin_metadata={},
+            timestamp_type=_log_timestamp_type_to_conjure(timestamp_type),
+        )
+        # we need to create the logset, then attach logs and finalize it
```

Comment:
i know i wrote this comment, but kinda useless given that the next two lines are "create the log set, then attach and finalize"